app/main.ipynb

```diff
@@ -126,19 +130,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "array[[0, 1, 4, 7]].sum()"
```

Comment:
The task description for Task 4 mentions obtaining a sum of 27 from a subsequence of the array. However, the indices used here (0, 1, 4, 7) result in a sum of 27.0, which matches the requirement. Ensure this matches the task's expected output.